Dockerfile

```diff
@@ -30,16 +28,32 @@ WORKDIR /home/node/app
 # Copy built application
 COPY --from=builder /home/node/app/dist .
 
-ARG PORT=9080
-ARG MNEMONIC
-ARG ADDRESS
-
-ENV PORT ${PORT}
-ENV MNEMONIC ${MNEMONIC}
-ENV ADDRESS ${ADDRESS}
-
-RUN npm i swagger-ui-express
-
-EXPOSE ${PORT}
-
-CMD [ "node", "index.js" ]
+# Build-time arguments
+ARG NODE_ENV=production
+ARG NPM_CONFIG_LOGLEVEL=warn
+ARG DID_REGISTRAR_PORT=3000
+ARG DID_REGISTRARi_MNEMONIC
+ARG DID_REGISTRAR_PORT
+
+# Run-time environment variables
+ENV NODE_ENV ${NODE_ENV}
+ENV NPM_CONFIG_LOGLEVEL ${NPM_CONFIG_LOGLEVEL}
+ENV DID_REGISTRAR_PORT ${DID_REGISTRAR_PORT}
+ENV DID_REGISTRAR_MNEMONIC ${DID_REGISTRAR_MNEMONIC}
+ENV DID_REGISTRAR_PORT ${DID_REGISTRAR_PORT}
+
+# Install pre-requisites
+RUN chown -R node:node /home/node/app && \
+    apk update && \
+    apk add --no-cache bash ca-certificates && \
+    npm install swagger-ui-express
```

Comment:
@ankurdotb this is an external dependency, which doesn't work well when bundled with the package, it's works fine when kept outside and loaded like this